### PR TITLE
Refresh warnings

### PR DESCRIPTION
**Pull request by jsingle**
@paetling

### CODE REVIEW

File: dusty/daemon.py [hunk 1]
```diff
@@ -61,6 +66,7 @@ def _listen_on_socket(socket_path, suppress_warnings):
                     suppress_warnings |= payload['suppress_warnings']
                     logging.info('Received command. fn: {} args: {} kwargs: {}'.format(fn.__name__, args, kwargs))
                     try:
+                        _refresh_warnings()
```
Comment (paetling) [inline dusty/daemon.py]: why are you doing this?

Comment (jsingle) [inline dusty/daemon.py]: Well warnings never get changed after preflight, even if someone upgrades the version of one of boot2docker, nginx, etc.  which is confusing

Comment (paetling) [inline dusty/daemon.py]: if you update something installed and then want daemon to automatically know about it?

Comment (jsingle) [inline dusty/daemon.py]: Don't want it to keep saying "WARNING yo shit is out of date" when it's no longer out of date

Comment (paetling) [inline dusty/daemon.py]: yep. We are on the same page.

**Comment by paetling** [comment]
:ghost: :cow:

**Pull request merged**